src/ray/common/scheduling/scheduling_test.cc

```diff
@@ -640,7 +640,7 @@ TEST_F(SchedulingTest, TaskResourceInstancesTest) {
 
     TaskRequest task_req;
     vector<FixedPoint> pred_demands = {3. /* CPU */, 2. /* MEM */, 1.5 /* GPU */};
-    vector<bool> pred_soft = {false};
+    vector<bool> pred_soft = {false, false, false};
```

Comment:
Interestingly ASAN was not able to detect these out of bounds array accesses, but the test behavior changed enough that it failed.